### PR TITLE
Fix for issue #1082

### PR DESCRIPTION
**Pull request by NimitS1**
I have removed the line which said that this article targets CSS 2.
Also, I have added border related properties as they are usually frequently used. In the CSS3 related properties which I added, I have mentioned in the comments that the property is a CSS3 property.

### CODE REVIEW

File: css.html.markdown [hunk 1]
```diff
@@ -13,12 +13,11 @@ further development of browsers, fully visual web pages also became common.
 CSS is the standard language that exists to keep the separation between   
 the content (HTML) and the look-and-feel of web pages.
 
+The latest version of CSS is CSS3. Before using the CSS3 properties, check if your browser is compatible with CSS3.
```
Comment (levibostian) [inline css.html.markdown]: Can you say *how* to check your browser?

**Comment by levibostian** [comment]
Asked a question there. Help me out there and merge master and this is ready to merge!

File: css.html.markdown (hunk 1 above)
Comment (NimitS1) [inline css.html.markdown]: There is a link at the bottom to check the compatibility of your browser (caniuse.com). Hence I did not mention the same again.

Comment (levibostian) [inline css.html.markdown]: Ok. Now to merge master and we can merge this in.

**Comment by vendethiel** [comment]
@NimitS1 pretty please? :)

**Comment by vendethiel** [comment]
@NimitS1 ?

**Comment by NimitS1** [comment]
Hey I am sorry!
I don't have laptop access till 6th July. I won't be able to work on this
till then :-/

**Comment by vendethiel** [comment]
No problem! Just wanted to know :).

**Comment by NimitS1** [comment]
Hi, the bug in question in this PR is not applicable anymore. I will close this PR now as it will be a wrong context. 
For the border stylesheet examples that I squeezed in along with the bug fix, I will open a new PR. Hope it is correct

**Pull request closed without merging**